### ISSUE TITLE
Error while parsing

### ISSUE THREAD

**Issue by anuraagvaidya** [sarosia/mjpeg2jpegs#1]
I used an MJPEG stream from iSpy server. The stream was playing properly on chrome and firefox.

Thought of letting you know the error:

```
events.js:141 throw er; // Unhandled 'error' event
        ^
Error: Unexpected content-type header multipart/x-mixed-replace; boundary=--myboundary
      at ClientRequest.<anonymous>  at ClientRequest.g (events.js:260:16)
  at emitOne (events.js:77:13)
at ClientRequest.emit (events.js:169:7)
at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:415:21)
at HTTPParser.parserOnHeadersComplete (_http_common.js:88:23)
at Socket.socketOnData (_http_client.js:305:20)
at emitOne (events.js:77:13)
at Socket.emit (events.js:169:7)
at readableAddChunk (_stream_readable.js:146:16)
```

**Comment by owenandrews** [comment]
Replacing line 9 with this worked for me (the \s? is the important part):
`var match = contentType.match(/multipart\/x-mixed-replace;\s?boundary=(.*)/);`

**Comment by sarosia** [comment]
Thanks for the comments. Fixed by #2.
